deliver/lib/deliver/runner.rb

```diff
@@ -72,10 +72,10 @@ def upload_metadata
     def prepare_app_icons(options = {})
       return unless options[:metadata_path]
 
-      default_app_icon_path = File.join(options[:metadata_path], "app_icon.png")
+      default_app_icon_path = Dir["#{options[:metadata_path]}/app_icon.{png,jpg}"].first
```

Comment:
`fastlane deliver download_metadata` and `fastlane deliver download_screenshots` is used by people, when the marketing team updates the metadata, and you want your repo to reflext those changes.